### DISCUSSION TITLE
how to change table system.settings

### DISCUSSION THREAD

**Discussion by flyly0755** [ClickHouse/ClickHouse#33972] [Q&A] [answered]
I want to change the value of background_pool_size in table system.settings from default vaule 16 to 32,
use set background_pool_size = 32; this command to change it,
then use sql language  select * from system.settings where name = 'background_pool_size'; to check it:
┌─name─────────────────┬─value─┬─changed─┬
│ background_pool_size │                      32    │       1 │ 
└──────────────────────┴───────┴─────────┴
show it changed successfully
but it don't really validate.
after quit the clickhouse-client and open clickhouse-client to check it again, it change back to 16 :(
it's no matter with restart clickhouse-server, after changed it i try to restart clickhouse-server,
after restart successfully i check it, it still is 16.

**Comment by zhongyuankai** [comment, marked as answer]
Need to make changes in the user.xml file and restart the service.
```
<profiles>
        <default>
            <background_pool_size>32</background_pool_size>
        </default>
</profiles>
```
